packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -159,7 +160,18 @@ export class SummarizerNode implements IRootSummarizerNode {
 			}
 		}
 
-		const result = await this.summarizeInternalFn(fullTree, true, telemetryContext);
+		const incrementalContext: IIncrementalContext = {
+			sequenceNumber: this.wipReferenceSequenceNumber ?? this._changeSequenceNumber,
+			previousSequenceNumber: this._latestSummary?.referenceSequenceNumber ?? 0,
+			parentPath: this._latestSummary?.fullPath.path ?? "",
+		};
+
+		const result = await this.summarizeInternalFn(
+			fullTree,
```

Comment:
Can we add some tests that would validate successful and failed paths and ensure that values passed to DDS are what we expect them to be?

Reply:
I initially tried end to end tests, but it became prohibitively expensive to create a DDS that would check the summaries. Our test infrastructure does not look kindly on making a new DDS. I tried an assert, but it broke a few tests. I'll have to do some custom work.